bpf/lib/maps.h

```diff
@@ -310,6 +310,17 @@ static __always_inline int lpm4_egress_lookup(__be32 addr)
 #define lpm4_egress_lookup(ADDR) 0
 #endif
 
+#if defined POLICY_EGRESS && defined LXC_ID
+/* Global IP -> Identity map for applying egress label-based policy */
+struct bpf_elf_map __section_maps cilium_remote_lxc = {
+	.type		= BPF_MAP_TYPE_HASH,
+	.size_key	= sizeof(struct endpoint_key),
+	.size_value	= sizeof(struct remote_endpoint_info),
+	.pinning	= PIN_GLOBAL_NS,
+	.max_elem	= ENDPOINTS_MAP_SIZE, /* XXX: Consider resizing? */
```

Comment:
I think it's fine but bumping it to 250K doesn't hurt either. The map is still quite small. Please create a GH to track adjusting this on demand. Given regenerations, we could easily resize the map.

Reply:
Created #3075. Main reason I put this comment was that it's now referring to remote containers so binding it to the same limit as the lxcmap might not be appropriate.